Library/Homebrew/test/cask/cmd/info_spec.rb

```diff
@@ -155,6 +173,8 @@
       Docker CE
       ==> Artifacts
       Docker.app (App)
+      ==> Analytics
+      install: 16,997 (30 days), 54,056 (90 days), 239,482 (365 days)
```

Comment:
There numbers seem like they should match those in the `expect` call?

Reply:
i tried with above, the mocking is happening but it cant able to parse information from the mocked data.

Reply:
sorry, forgot to parse the data :)